test/src/unittests/spectral/test_logspectrum.py

```diff
@@ -18,88 +18,287 @@
 # version 3 along with this program. If not, see http://www.gnu.org/licenses/
 
 
-
 from essentia_test import *
 import numpy as np
 
-class TestLogSpectrum(TestCase):
 
+class TestLogSpectrum(TestCase):
     def testRegression(self, frameSize=8192 + 1):
```

Comment:
There's lots of formatting change that break our current (though poorly consistent) style. I would prefer to only keep changes in the content of the tests for this PR.